odpf/meta/facets/Columns.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package odpf.meta.facets;
+
+option java_package = "io.odpf.proton.meta.facets";
+option java_outer_classname = "Columns";
+option go_package = "github.com/odpf/proton/meta/facets";
+
+message Columns {
+  repeated Column columns = 1;
+}
+
+message Column {
+  string name = 1;
+  string description = 2;
+  string data_type = 3;
+  bool is_nullable = 4;
+  int64 length = 5;
```

Comment:
what are we storing in length?

Reply:
for `varchar` data type can be something like `255`.